edge/pkg/metamanager/processor/common_processor.go

```diff
@@ -0,0 +1,279 @@
+package processor
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"k8s.io/klog/v2"
+
+	beehiveContext "github.com/kubeedge/beehive/pkg/core/context"
+	"github.com/kubeedge/beehive/pkg/core/model"
+	"github.com/kubeedge/kubeedge/common/constants"
+	"github.com/kubeedge/kubeedge/edge/pkg/common/modules"
+	metaManagerConfig "github.com/kubeedge/kubeedge/edge/pkg/metamanager/config"
+	"github.com/kubeedge/kubeedge/edge/pkg/metamanager/dao"
+)
+
+//Constants to check metamanager processes
+const (
+	OK = "OK"
+
+	GroupResource     = "resource"
+	OperationMetaSync = "meta-internal-sync"
+
+	OperationFunctionAction = "action"
+
+	OperationFunctionActionResult = "action_result"
+
+	EdgeFunctionModel   = "edgefunction"
+	CloudFunctionModel  = "funcmgr"
+	CloudControlerModel = "edgecontroller"
+)
+
+type insertProcessor struct{}
+
+type updateProcessor struct{}
+
+type deleteProcessor struct{}
+
+type queryProcessor struct{}
+
+type responseProcessor struct{}
```

Comment:
It is recommended that the method follow the `struct` to increase readability

Reply:
done